internal/api_server/agentserver/server.go

```diff
@@ -18,7 +19,6 @@ import (
 	service "github.com/kubev2v/migration-planner/internal/service/agent"
 	"github.com/kubev2v/migration-planner/internal/store"
 	"github.com/kubev2v/migration-planner/internal/util"
-	"github.com/leosunmo/zapchi"
```

Comment:
I think we can now remove it from `go.mod`

Reply:
Done